### PR TITLE
Start to add JSDoc + fix CSS errors

### PR DESCRIPTION
**Pull request by archiloque**

### CODE REVIEW

File: gleam-editor.js [hunk 1]
```diff
@@ -66,11 +77,14 @@ class FauxRadioSet {
...
 }
 
-// Very basic overlay handling
-// TODO maybe the overlay should be an object.
-// TODO maybe the overlay should be able to operate as a promise
-// TODO the overlay should be able to position itself by the mouse cursor, if opened in response to a click
-// TODO a transient overlay should probably disappear on document blur?
+/**
+ * Very basic overlay handling
+ * TODO maybe the overlay should be an object.
```
Comment (eevee) [inline gleam-editor.js]: is it appropriate to include TODOs in the jsdoc?  doesn't that make them part of the documentation?

Comment (archiloque) [reply gleam-editor.js]: I guess it depends if you want to make them visible or not, what do you prefer ? I can change to move them out of JSDoc if you want.

Comment (eevee) [reply gleam-editor.js]: yeah most of them are half-baked thoughts about implementation details that wouldn't make much sense to someone reading the docs, so they should be left out.  i don't know where they can go without breaking the jsdoc association, though, so i'll have to leave that to you to undo.  after that i'll merge this before i go messing with everything

Comment (archiloque) [reply gleam-editor.js]: Right, I've moved the TODO and FIXME out of the JSDoc blocks, I've checked and they don't seem to appear in the diff anymore

**Pull request merged**